bin.src/make_remote_butler.py

```diff
@@ -97,46 +96,6 @@ def _add_chains(butler, instrument_name):
     )
 
 
-def _hsc_rc2(butler):
-    """fix up some specifics of the HSC-RC2 dataset export
-
-    Parameters
-    ----------
-    butler: `lsst.daf.butler.Butler`
-        The source Butler from which datasets are exported
-    """
-    # Chain calibration collections
-    instrument = Instrument.fromName("HSC", butler.registry)
-    butler.registry.setCollectionChain(
-        instrument.makeCalibrationCollectionName(),
-        [
-            "HSC/calib/DM-32378",
-            "HSC/calib/gen2/20180117",
-            "HSC/calib/DM-28636",
-        ],
-    )
```

Comment:
This fix up wasn't really needed in the past, because we can export the `HSC/calib` chain from `/repo/main`.   IIRC I was just too confused back then, or I couldn't figure out the correct parameters for `flattenChains` and `includeChains`, and using a fix-up seemed easier back then.

Reply:
Also, at the time we didn't have the (HSC/)templates collection set up in `/repo/main`, so some kind of fix-up was necessary back then anyway.